_build/data/transport.core.system_settings.php

```diff
@@ -1073,6 +1073,15 @@
   'area' => 'language',
   'editedon' => null,
 ], '', true, true);
+$settings['package_installer_at_top'] = $xpdo->newObject('modSystemSetting');
+$settings['package_installer_at_top']->fromArray(array (
+  'key' => 'package_installer_at_top',
+  'value' => true,
+  'xtype' => 'combo-boolean',
+  'namespace' => 'core',
+  'area' => 'manager',
+  'editedon' => null,
+), '', true, true);
```

Comment:
The system setting class needs to be specified in this way, as otherwise a fatal error is thrown when running `_build/core.transport.php`. Also, we no longer use the `array` keyword and prefer the short syntax.
```suggestion
$settings['package_installer_at_top'] = $xpdo->newObject(modSystemSetting::class);
$settings['package_installer_at_top']->fromArray([
  'key' => 'package_installer_at_top',
  'value' => true,
  'xtype' => 'combo-boolean',
  'namespace' => 'core',
  'area' => 'manager',
  'editedon' => null,
], '', true, true);
```